README.md

```diff
@@ -1,31 +1,116 @@
+
+> **Note**: We are in public beta and welcome early adopter feedback!  
+> Feel free to reach out [via Discord] or open [issues in GitHub].
+>
+> [issues in github]: https://github.com/iterative/vscode-dvc/issues
+> [via discord]: https://discord.gg/BGCjJHvDHt
+
+### Why prefer this extension?
+
+- Enjoy the best developer experience with the first experiment tracking
+  interface for an IDE.
+- No external servers, databases, subscriptions, etc. Data stays fully under
+  your control and your existing Git hosting is used to share and collaborate.
+- Go beyond a simple metrics dashboard with complete ML experiments that include
+  metrics, code, and data. Powered by [DVC experiment
+  versioning][dvc experiments].
+- Implement [data versioning] on top of your favorite cloud storage, such as
+  Amazon S3, Azure Blob Storage, Google Cloud Storage, NFS, etc.
+
+[dvc experiments]:
+  https://dvc.org/doc/user-guide/experiment-management/experiments-overview
+[data versioning]: https://dvc.org/doc/use-cases/versioning-data-and-model-files
+
+## Quick start
+
+1. [Install DVC](https://dvc.org/doc/install) on your system.
+2. Install [this extension] in VS Code.
+3. Follow the **Get Started** page that pops up!
+
+<!-- prettier-ignore -->
+> 💡 Feel free to try our [example DVC project] first! Opening it [with Github
+> Codespaces] will include this extension automatically.
+
+![Initial Get Started page](https://raw.githubusercontent.com/iterative/vscode-dvc/main/extension/docs/walkthroughs.png)
+
+[this extension]:
+  https://marketplace.visualstudio.com/items?itemName=Iterative.dvc
+[example dvc project]: https://github.com/iterative/example-get-started
+[with github codespaces]:
+  https://docs.github.com/en/codespaces/getting-started/quickstart#creating-your-codespace
+
+### Learn more about DVC (Data Version Control)
+
+See the DVC documentation to Get Started with [Experiment Versioning] or [Data
+Management]. For deeper learning, try our [free course]!
+
+[experiment versioning]: https://dvc.org/doc/start/experiments
+[data management]: https://dvc.org/doc/start/data-management
+[free course]: https://learn.iterative.ai/
+
+<!-- [learn more]: extension/resources/walkthrough/dvc-learn-more.md -->
+
+## What you get
+
+This extension augments VS Code in the following ways:
+
+- Adds a **[DVC Tracked]** panel to the _Explorer_ view. This shows a tree of
+  the data files and directories tracked by DVC, including their state, and
+  options to [synchronize] them (from/to remote storage).
+- Adds a **[DVC panel]** to the _Source Control_ view to display the [workspace
+  status]. You can [restore] or reset project versions (based on the current Git
+  `HEAD` commit) as well as manage new and existing data from here.
+- Adds a brand-new new **[DVC View]** (<img
+  src="https://raw.githubusercontent.com/iterative/vscode-dvc/main/extension/docs/dvc.png"
+  alt="DVC logo" style="height: 1em;"/> icon in the Activity Bar) with panels to
```

Comment:
![image](https://user-images.githubusercontent.com/37993418/173559355-9ae0eea9-7279-4297-97bb-11e55b282138.png) looks like this styling doesn't work for the in-app content

Reply:
fixed this by changing from style to height.

Reply:
Thanks